packages/ifpack2/src/Ifpack2_Relaxation_decl.hpp

```diff
@@ -782,6 +782,8 @@ class Relaxation :
   bool fixTinyDiagEntries_;
   //! Whether to spend extra effort and all-reduces checking diagonal entries.
   bool checkDiagEntries_;
+  //! For MTSGS, the cluster size (use point coloring if equal to 1)
+  int clusterSize_;
```

Comment:
At some point (not now), we should start using member initializers to set defaults.